packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/representation/DiagramRepresentation.types.ts

```diff
@@ -12,11 +12,12 @@
  *******************************************************************************/
 
 import { GQLDiagramEventPayload } from '../graphql/subscription/diagramEventSubscription.types';
-import { Diagram } from '../renderer/DiagramRenderer.types';
+import { Diagram, DiagramDescription } from '../renderer/DiagramRenderer.types';
 
 export interface DiagramRepresentationState {
   id: string;
   diagram: Diagram | null;
+  diagramDescription: DiagramDescription | null;
```

Comment:
It seems you are just using `autoLayout`, why do you need the whole diagram description here?
Maybe `autoLayout` could be enough.